src/tutorials/media_library.md

```diff
@@ -0,0 +1,221 @@
+---
+id: media_library
+name: Uploading Images
+title: Uploading Images
+tags:
+    - Tutorial
+---
+<!-- vale Manticore.FirstPerson = NO -->
+# Uploading Images using the Media Library
+
+![!Preview](../img/MediaLibraryTutorial/preview.png){: .center loading="lazy" }
+
+## Overview
+
+In this tutorial, you will learn how to use the [Media Library](../references/media_library.md) in Core to upload images to your account from within the Core editor that can be used in any of your projects.
+
+Being able to upload images allows you to create UI (User Interface) that is unique for your game that can be used in screen space and world space which can help make your game stand out and improve the player experience.
+
+The focus of this tutorial will be the Media Library and creating brushes from images you have uploaded to the Media Library. Interactive functionality will not be covered, for example, making buttons the player can click on.
+
+* **Completion Time:** ~20 minutes
+* **Knowledge Level:** No previous knowledge.
+* **Skills you will learn:**
+    * Where to find the Media Library.
+    * How to upload images to the Media Library.
+    * Creating image brushes from content in your Media Library.
+    * Using image brushes in the UI to create a shop.
+
+## Download Image Pack
+
+For this tutorial, you can use your images if you prefer, or if you want to follow along to recreate the shop UI, then you will need to download an image pack such as the one linked below. The pack below will require you to cut or slice the parts you want to use.
```

Comment:
mention creating a shop UI before offering the options 
your *own
cut "if you prefer"
cut "then you will need to" and "such as"